src/scripts/settings.js

```diff
@@ -642,9 +606,19 @@ document.addEventListener('DOMContentLoaded', function(e) {
     );
     Settings.$sendErrorReports = document.querySelector('#send-error-reports');
 
+    if (FF) {
```

Comment:
maybe add a comment here to indicate it has to do with hiding the permissions tab.